libs/damap/src/lib/components/dmp/data-deletion/data-deletion.component.html

```diff
@@ -3,33 +3,38 @@
     <div class="warning-icon">
       <mat-icon>error_outline</mat-icon>
     </div>
-    {{'dmp.steps.data.deletion.warning.published' | translate}}
+    {{ 'dmp.steps.data.deletion.warning.published' | translate }}
   </div>
   <div>
-    <p>{{'dmp.steps.data.deletion.question.label' | translate}}</p>
-    <mat-slide-toggle formControlName="delete"
-                      color="primary">{{'dmp.steps.data.deletion.question.delete' | translate}}</mat-slide-toggle>
+    <p>{{ 'dmp.steps.data.deletion.question.label' | translate }}</p>
+    <mat-slide-toggle formControlName="delete" color="primary">{{
+      'dmp.steps.data.deletion.question.delete' | translate
+      }}</mat-slide-toggle>
   </div>
   <div *ngIf="dataset.value.delete">
     <div class="form-row">
       <mat-form-field class="full-width">
-        <mat-label>{{'dmp.steps.data.deletion.question.person' | translate}}</mat-label>
+        <mat-label>{{
+          'dmp.steps.data.deletion.question.person' | translate
+          }}</mat-label>
         <mat-select formControlName="deletionPerson" [compareWith]="getSelection">
           <mat-option [value]="null"></mat-option>
           <mat-option *ngFor="let contributor of contributors" [value]="contributor">
-            {{contributor.firstName + ' ' + contributor.lastName}}
+            {{ contributor.firstName + ' ' + contributor.lastName }}
           </mat-option>
         </mat-select>
       </mat-form-field>
       <mat-form-field>
-        <mat-label>{{'dmp.steps.data.deletion.question.date' | translate}}</mat-label>
-        <input matInput formControlName="dateOfDeletion" [matDatepicker]="picker">
+        <mat-label>{{
+          'dmp.steps.data.deletion.question.date' | translate
+          }}</mat-label>
+        <input matInput formControlName="dateOfDeletion" [matDatepicker]="picker" [(ngModel)]="planModel.end_time" />
```

Comment:
Using `[(ngModel)]` like this overwrites the form value everytime the form is loaded. Also `formControlName` and [(ngModel)] should not be used on the same form field since they represent two different ways of working with forms, but access the same field (see also: https://angular.io/guide/forms-overview). 
It's probably better to set the `dateOfDeletion` to the project end date when `delete` is changed (using the event emitted by toggle to distinguish user changes from programmatic changes).
(Can be done by patching the value of a given formgroup - it should look something like this and can be done in the `component.ts`: https://github.com/tuwien-csd/damap-frontend/blob/next/libs/damap/src/lib/services/form.service.ts#L291)

Reply:
Ill check this now.